package.json

```diff
@@ -16,9 +16,10 @@
     "url": "https://github.com/Microsoft/react-native-code-push"
   },
   "dependencies": {
-    "code-push": "^1.1.1-beta"
+    "code-push": "^1.1.1-beta",
+    "semver": "^5.1.0"
```

Comment:
Yeah good call on this dependency, v-flexible to different formats that conform to semver